opentracing/harness/api_check.py

```diff
@@ -126,6 +126,8 @@ def test_span_logs(self):
             log_event('defrosted', {'year': 2999}). \
             log_event('became his own grandfather', 1947)
         span.\
+            log(event='frozen'). \
```

Comment:
argh, thank you... I wrote that other thing in a hurry and it showed. too much javascript lately :-/